Lib/encodings/__init__.py

```diff
@@ -61,7 +61,8 @@ def normalize_encoding(encoding):
         if c.isalnum() or c == '.':
             if punct and chars:
                 chars.append('_')
-            chars.append(c)
+            if c.isascii():
+                chars.append(c)
```

Comment:
I wanted to ask you to add a ".. versionchanged:: 3.10" entry in the documentation, but then I noticed that the encodings module was never documented! Oh!

Reply:
If end user will use this function or module, I can try to create the doc, but I need some time to do it :)

Reply:
It can and must be addressed in a separated PR anymore. The lack of documentation should not hold this change.

Reply:
ok, copy that.